style/text-viz.less

```diff
@@ -14,6 +14,49 @@
         font: @text-value-size @text-value-font-family;
         color: @label-color;
         line-height: @text-value-line-height;
+        
+        span{
```

Comment:
Yep, it will just animate to the new values of top/bottom, opacity, colour and transform